android/playground/src/main/java/com/kiwi/mobile/rnandroidplayground/AncillaryFactoryFragment.kt

```diff
@@ -0,0 +1,10 @@
+package com.kiwi.mobile.rnandroidplayground
+
+import com.facebook.react.ReactNativeHost
+import com.kiwi.rnkiwimobile.ancillaries.RNAncillaryFactoryFragment
+import com.kiwi.rnkiwimobile.ancillaries.RNAncillaryFactoryInitialProperties
+
+
+class AncillaryFactoryFragment: RNAncillaryFactoryFragment(RNAncillaryFactoryInitialProperties(service = "fast_track", bookingId = 123, kwAuthToken = "mock")) {
+    override fun getReactNativeHost(): ReactNativeHost = (activity?.application as PlaygroundApplication).reactNativeHost
+}
```

Comment:
add empty line on the end of the file. Setup you editor to do that automatically...

Reply:
Fixed, and finally set editor:)